test/statetest/statetest.hpp

```diff
@@ -59,8 +63,52 @@ struct StateTransitionTest
     std::unordered_map<uint64_t, std::string> input_labels;
 };
 
+template <typename T>
+T from_json(const json::json& j) = delete;
+
+template <>
+state::BlockInfo from_json<state::BlockInfo>(const json::json& j);
+
+template <>
+state::State from_json<state::State>(const json::json& j);
+
+template <>
+state::Transaction from_json<state::Transaction>(const json::json& j);
+
 StateTransitionTest load_state_test(const fs::path& test_file);
 
 void run_state_test(const StateTransitionTest& test, evmc::VM& vm);
 
+namespace utils
+{
+
+inline std::string shortest_hex(uint64_t v, bool add_prefix = true)
```

Comment:
Is this really needed? I believe `intx::hex` is already producing the "shorted" hex as expected for any integer.

Reply:
you are right. Fixed.

Reply:
In fairness, we slowly arrived at the point of using `intx::hex()` which then allowed this further simplification 😅